kani-driver/src/args.rs

```diff
@@ -343,6 +347,16 @@ pub enum ConcretePlaybackMode {
     InPlace,
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq, ValueEnum, strum_macros::Display)]
+#[strum(serialize_all = "kebab-case")]
+pub enum UnstableFeatures {
+    /// Allow replacing certain items with stubs (mocks).
+    /// See [RFC-0002](https://model-checking.github.io/kani/rfc/rfcs/0002-function-stubbing.html)
+    Stubbing,
+    /// Generate a C-like file equivalent to input program used for debugging purpose.
+    GenC,
```

Comment:
Are we planning to hook these up in a follow-up PR?

It might also be worth adding a variant for concrete playback.

Reply:
Yes, we need to move all the unstable features to this new model and deprecate the old `--enable-unstable`.